central/complianceoperator/v2/pipelines/complianceoperatorinfo/pipeline.go

```diff
@@ -16,6 +17,8 @@ import (
 	"github.com/stackrox/rox/pkg/metrics"
 )
 
+const minimalComplianceOperatorVersion = "v1.6.0"
```

Comment:
Maybe make it an env var? What if the release is out and we suddenly discover that the hardcoded version 123 is not the minimal required but instead 234 should be the minimum?

Reply:
> In that case, anyone can set the environment to whatever they want.

This is not a good argument against that, as we have many places in the code when customers can break things if they deliberately set the wrong values.

I understand that you want to accept the risk and don't want to implement this. This is fine, but please always consider that some other colleagues maybe maintaining this while you are on holidays and their risk assessment may be different. I shared mine with you.